lib/statsd.rb

```diff
@@ -14,19 +14,82 @@
 #   statsd = Statsd.new('localhost').tap{|sd| sd.namespace = 'account'}
 #   statsd.increment 'activate'
 class Statsd
-  class RubyUdpClient
-    attr_reader :key, :sock
+  class BufferedUDPClient < UDPClient
+    @max_packet_size = 512
```

Comment:
Piling on late here, but if this is really just the max size that will be batched perhaps the name is a little confusing. Maybe max_batch_size rather than packet? Since we could send out a packet that's larger if someone sends us a huge key or something (by accident.)